build_tools/cmake/iree_check_test.cmake

```diff
@@ -57,7 +57,7 @@ function(iree_check_test)
     _RULE
     ""
     "NAME;SRC;TARGET_BACKEND;DRIVER;MODULE_FILE_NAME"
-    "COMPILER_FLAGS;RUNNER_ARGS;LABELS;TARGET_CPU_FEATURES;TIMEOUT"
+    "COMPILER_FLAGS;RUNNER_ARGS;LABELS;TARGET_CPU_FEATURES;DATA;TIMEOUT"
```

Comment:
same comment about `DATA` as in `iree_bytecode_module.cmake`.

Reply:
It should be called `DEPS` not `DATA`